packages/sign-client/src/controllers/engine.ts

```diff
@@ -868,11 +942,15 @@ export class Engine extends IEngine {
     });
 
     return {
-      uri: connectionUri,
+      uri: linkModeUri ?? connectionUri,
```

Comment:
@ganchoradkov what do you think of this? in link-mode i return the URL to redirect.

Reply:
the return of this fx is used for QR code generation so will have to test how it behaves

Reply:
The QR code should work, as in that case we don't send a wallet universal link as param so it generates an URI.

But it will break the `Copy Link` button as it will copy the URL

<img height=700 src="https://github.com/WalletConnect/walletconnect-monorepo/assets/25931366/82b78513-7057-4b73-adae-8254f66c392d" />